### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/07/day6.html

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#15] [Comments]
# https://mnfienen.github.io/talulat/2023/08/07/day6.html

Day 6 - Passing of the storm miles today: 41 total miles: 326 km today: 66 total km: 534

https://mnfienen.github.io/talulat/2023/08/07/day6.html

**Comment by aleaf** [comment]
#onelesscar

**Comment by christianlangevin** [comment]
Awesome, Mike!

**Comment by SmittysGitHub** [comment]
A little surprised by the struggle with rock names or did you just really nail it in the periodic table?

**Reply by mnfienen** [reply]
honestly, I've been focused on water science for so long that my rock naming skills have atrophied immensely ...
